kratos/includes/kernel.h

```diff
@@ -109,8 +109,17 @@ class KRATOS_API(KRATOS_CORE) Kernel {
     */
     void InitializeApplication(KratosApplication& NewApplication) {}
 
-    bool IsImported(std::string ApplicationName) const;
-
+    /**
+     * @brief This method returns if the given application by name is imported
+     * @param rApplicationName The name of the application of interest
+     * @return True if imported, false otherwise
+     */
+    bool IsImported(const std::string& rApplicationName) const;
```

Comment:
I think this can be static

Reply:
Nope, it cant' I already tried

Reply:
well `GetApplicationsList` also needs to be const then
I am not sure why it is not
`application_list` is static

Reply:
This was already like this, I only added the documentation

Reply:
I know
But can you please try making them static? I think it would be useful in the future

Reply:
I already tried, and gave me compilation errors

Reply:
hmmm
Ok I would like to try myself (and try sth else too)